### PR TITLE
Add interactive icons tokens

### PR DESCRIPTION
**Pull request by NotNestor**
Screenshot 2021-08-24 at 22 00 35

### CODE REVIEW

File: components/interactive-icon.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      },
+      "#pressed": {
+        "background": "@theme-button-secondary-pressed",
+        "text": "@theme-text-primary-normal"
+      },
+      "#disabled": {        
+        "text": "@theme-text-primary-disabled"
+      }      
+    },
+    "attatchment": {
```
Comment (paulwitty) [inline components/interactive-icon.json:57]: Spelling!

File: components/interactive-icon.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      },
+      "#pressed": {
+        "background": "@theme-button-secondary-pressed",
+        "text": "@theme-text-primary-normal"
+      },
+      "#disabled": {        
+        "text": "@theme-text-primary-disabled"
+      }
+    },
+    "attatchment-active": {
```
Comment (paulwitty) [inline components/interactive-icon.json:74]: And again.
I suggest we change it to 
"attachment": {
  "#normal": {
    "icon-active": "@theme-text-accent-normal",
    "icon-inactive": "@text-theme-primary-normal",
    "background": "@theme-button-secondary-normal"
  },
  "#hovered": {
   "background": "@theme-button-secondary-hover"
  }

File: components/interactive-icon.json (hunk 1 above)
Comment (NotNestor) [reply components/interactive-icon.json:57]: I think this is my most commonly misspelled word

File: components/interactive-icon.json (hunk 2 above)
Comment (NotNestor) [reply components/interactive-icon.json:74]: 👍

**Review by paulwitty — APPROVED** [review]

**Review by paulwitty — CHANGES_REQUESTED** [review]
Changed my mind...

**Review by paulwitty — APPROVED** [review]

**Pull request merged**